src/ACEtk/block/TabulatedOutgoingEnergyDistribution.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef NJOY_ACETK_BLOCK_TABULATEDOUTGOINGENERGYDISTRIBUTION
+#define NJOY_ACETK_BLOCK_TABULATEDOUTGOINGENERGYDISTRIBUTION
+
+// system includes
+
+// other includes
+#include "ACEtk/block/details/TabulatedProbabilityDistribution.hpp"
+
+namespace njoy {
+namespace ACEtk {
+namespace block {
+
+/**
+ *  @class
+ *  @brief Tabulated secondary energy distribution data from the DLW block for
+ *         a single reaction and incident energy
+ *
+ *  The TabulatedOutgoingEnergyDistribution class contains the probability
+ *  density function (PDF) and cumulative density function (CDF) as a function
+ *  of the outgoing energy for a single incident energy.
+ */
+class TabulatedOutgoingEnergyDistribution :
+  protected details::TabulatedProbabilityDistribution {
+
+  /* fields */
+  double incident_;
```

Comment:
Not all constructors guarantee this is set. Suggest adding a default value.

Reply:
Right. This should be only an issue with the default constructor, right?

I only define those because the python bindings need a default constructor when a variant is used (in ENDFtk I used to wrap these in precompiler statements but I didn't do that here).